src/codemagic/tools/_app_store_connect/actions/publish_action.py

```diff
@@ -383,7 +391,7 @@ def _process_ipa_after_upload(
                 **testflight_options.__dict__,
                 'max_build_processing_wait': 0,  # Overwrite waiting since we already waited above.
             }
-            self.submit_to_testflight(build.id, **testflight_submission_kwargs)
+            self.submit_to_testflight(build.id, **testflight_submission_kwargs)  # type: ignore
```

Comment:
This can be done in a type-safe manner.

```python
        if testflight_options:
            # Overwrite waiting since we already waited above.
            tf_options = dataclasses.replace(testflight_options, max_build_processing_wait=0)
            self.submit_to_testflight(build.id, **tf_options.__dict__)
```